rerun_py/rerun_sdk/rerun/archetypes/text_log.py

```diff
@@ -56,11 +57,27 @@ def __init__(
         """Create a new instance of the TextLog archetype."""
 
         # You can define your own __init__ function as a member of TextLogExt in text_log_ext.py
-        self.__attrs_init__(text=text, level=level, color=color)
+        with catch_and_log_exceptions(context=self.__class__.__name__):
+            self.__attrs_init__(text=text, level=level, color=color)
+            return
+        self.__attrs_init__(
+            text=None,
+            level=None,
+            color=None,
+        )
+
+    @classmethod
+    def _clear(cls) -> TextLog:
```

Comment:
The clear-method is basically what we discussed in https://github.com/rerun-io/rerun/issues/3381 - maybe we should just make it public (call it `clear`)?

Reply:
It's close, but I was worried about situations where it might still fail. It's still sort of a "try_clear_as_fallback".